### PR TITLE
Jazzy add icon #18

### LINKED ISSUE #18: Add 'Add icon' to the homepage
- This needs to go on the homepage
- Reference wireframe
- Make it circular with a plus icon in it
- Connect this icon to the 'Transaction' screen/page

### PR DESCRIPTION
**Pull request by Singh2je**
## Proposed Changes
- Created 2 different views for the Home Fragment - homePageView and popUpTransactionView
- On click of the Add button, it makes the Transaction view visible, and when save button is clicked, it makes the homePageView Visible.

## Checklist
- [x] You have added unit tests
- [x] You've added documentation
- [x] You've assigned reviewers 
- [x] You've assigned assignees 
- [x] Your code builds clean without any errors or warnings
- [x] You are following coding conventions

### CODE REVIEW

File: .idea/gradle.xml [hunk 1]
```diff
@@ -15,6 +15,7 @@
           </set>
         </option>
         <option name="resolveModulePerSourceSet" value="false" />
+        <option name="useQualifiedModuleNames" value="true" />
```
Comment (sukhbirsekhon) [inline .idea/gradle.xml:18]: Do we really need this?

**Review by sukhbirsekhon — CHANGES_REQUESTED** [review]
I added two comments. Please fix them and I'll merge your PR

File: app/src/main/java/home/project/group/financetracker/ui/home/HomeFragment.java [hunk 2]
```diff
@@ -49,8 +76,44 @@ public View onCreateView(@NonNull LayoutInflater inflater,
...
+          View myView = view.findViewById(R.id.navigation_transaction);
+                myView.performClick();
+
+   /*   MenuItem menuItem = (MenuItem) view.findViewById(R.id.navigation_transaction);
+                menuItem.setActionView(R.id.navigation_transaction);
+
+    MenuItem actionRestart = (MenuItem) view.findViewById(R.id.navigation_transaction);
+                actionRestart.getActionView();*/
+
+
```
[the remark this answers is not in the document]
Comment (Singh2je) [reply app/src/main/java/home/project/group/financetracker/ui/home/HomeFragment.java]: Removed extra spaces and comments

File: .idea/gradle.xml (hunk 1 above)
Comment (Singh2je) [reply .idea/gradle.xml]: I removed it and it works fine.

**Pull request merged**